Source/WebCore/platform/graphics/texmap/TextureMapperLayer.cpp

```diff
@@ -448,6 +448,7 @@ void TextureMapperLayer::collectDamageSelf(TextureMapperPaintOptions& options, D
         m_receivedDamage = { };
         m_inferredLayerDamage = { };
         m_inferredGlobalDamage = { };
+        m_damageCollectedAtLeastOnce = true;
```

Comment:
Is there a way we could change this variable name so instead of saying _what_ ("damage has been collected at least once"), it said why or when? 

> This change fixes the damage inference logic in a way that newly added
TextureMapperLayers are not taking the damage from before transform
change into account.

This is the _why_ -> we want to make sure the transform is taken into account. That reason can easily be forgotten after a while, and the variable name won't convey it